### DISCUSSION TITLE
How to get new Java Jar

### DISCUSSION THREAD

**Discussion by youngmip** [rdkit/rdkit#6688] [Q&A] [answered]
Hi, 

Can you please help me where to get the latest Jar file?
![image](https://github.com/rdkit/rdkit/assets/11573733/969a0d40-5808-47e4-8666-c81d6530e3d1)

I created from https://github.com/rdkit/rdkit/releases/tag/Release_2023_03_3 following the documentation, but  I can't find the RegistrationHash class and the RDKFuncs.RDKFuncsInit() method 

As @ptosco suggested, I used the Jar file from this location but I still have the same problem
https://github.com/rdkit/rdkit-build-helpers/tree/main/swig_wrappers/artifacts/java/noarch

Can you please get advice? 

Thanks a lot
Youngmi

**Comment by ptosco** [comment, marked as answer]
`RegistrationHash` is currently implemented in the Python layer, not in the C++ layer. As the Java wrappers are built from the C++ layer, `RegistrationHash` is not available from Java, as explained [here](https://www.mail-archive.com/rdkit-discuss@lists.sourceforge.net/msg11438.html).

**Reply by youngmip** [reply]
Thank you for the clarification.
